src/rebar_digraph.erl

```diff
@@ -123,3 +123,7 @@ find_app_by_name(Name, Apps) ->
     ec_lists:find(fun(App) ->
                           rebar_app_info:name(App) =:= Name
                   end, Apps).
+
+all_apps_deps(App) ->
+    Applications = [atom_to_binary(X, utf8) || X <- rebar_app_info:applications(App)],
+    lists:usort(rebar_app_info:deps(App) ++ Applications).
```

Comment:
And the `usort` isn't actually necessary, I just figured why not make the amount of nodes in the digraph minimal.